distro/pom.xml

```diff
@@ -375,8 +375,24 @@
                   </replacevalue>
                 </replace>
 
+                <echo level="info" message="Adding traefik server stop and destroy script" />
+                <replace
+                  file="${project.build.directory}/${project.artifactId}-${project.version}/run/docker/scripts/destroy-demo.sh"
+                  token='docker compose -p $PROJECT_NAME $dockerComposeProxyCLIOptions down -v'>
+                  <replacevalue>
+                    docker compose -p $PROJECT_NAME $dockerComposeProxyCLIOptions down -v&#10;&#9;echo "$INFO Destroying Traefik services..."&#10;&#9;docker compose -p $PROJECT_NAME $dockerComposeTraefikCLIOptions down -v
+                  </replacevalue>
+                </replace>
+                <replace
+                  file="${project.build.directory}/${project.artifactId}-${project.version}/run/docker/scripts/stop-demo.sh"
+                  token='docker compose -p $PROJECT_NAME $dockerComposeProxyCLIOptions stop'>
+                  <replacevalue>
```

Comment:
Does running `dockerComposeTraefikCLIOptions` only stop all containers? The command seems to be running  the stop twice

Reply:
how about now

Reply:
This resolves it.